spec/book_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe Book do
+  before :each do
+    @book = Book.new(title: 'Love', author: 'Ced')
+  end
+
+  describe '#new' do
+    it 'returns a new book object' do
+      @book.should be_an_instance_of Book
+    end
```

Comment:
- [ ] Since we're aiming to cover all the methods in our classes, kindly write a test for the `add_rental` method in your `book` class 👍